### ISSUE TITLE
[Adaptive Card iOS - Action.ShowCard.Mode.Json]: After activating the 'More Information' button, screen reader is not announcing the information which is appeared on the screen.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#181]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS Version: 16.7.1
App Version: 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**

1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.5" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'Action.ShowCard.Mode.Json' and activate it.
7. Perform right swipe and navigate to the 'Root Overflow Action' button.
8. Double tap to activate it.
9. Perform right swipe and navigate to the more option button.
10. Double tap on it and observe the issue.

**Actual Result:**
After activating the 'More Information' button, screen reader focus is neither announcing the information about the displayed info nor landing on it.

**Expected Result:**
After activating the 'More Information' button screen reader should announce the displayed information to the user.
Also, focus should be land on the displayed information.

**Note:**
Same issue is repro throughout the application for the same type of control.

**User Experience:**
It will impact the screen reader user if the voice over does not convey the complete information for the day.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/774ad096-877f-4009-9299-382faa549082

### Card JSON

```JSON
NA
```
